app/helpers/logger/logger.js

```diff
@@ -18,31 +18,35 @@ const Logger = function (debugging, externalLogger, fileSystem) {
 
   };
 
-  const error = function (detail, errorDetail) {
+  const getMessage = function (type, detail, fileUrl) {
+    return type + ": " + detail + ((fileUrl) ? " - " + fileUrl : "");
+  };
+
+  const error = function (detail, errorDetail, fileUrl, fileName) {
     let logDatetime = getLogDatetime();
-    let message = "ERROR: " + detail + " " + errorDetail;
+    let message = getMessage("ERROR", detail + " " + errorDetail, fileUrl);
```

Comment:
@stulees I think we should so we have it consistent and also we can have this information in case network is down and the log did not go to BQ. The message is used to be logged to the console and also to the file.